lib/registrant.rb

```diff
@@ -0,0 +1,18 @@
+class Registrant
+  attr_reader :name, :age, :license_data
+
+  def initialize(name, age, permit = false, license_data = false)
```

Comment:
We don't need the license_data in the arguments here, we can just set that within the intialize like you have